pages/Ethereum/Validation.md

```diff
@@ -0,0 +1,30 @@
+---
+menu: "DSNP: Ethereum"
+name: Announcement Validation
+route: /Ethereum/Validation
+---
+
+# Announcement Validation
+
+DSNP Announcements on Ethereum are validated at read time.
+
+1. Read `DSNPBatchPublication` events
```

Comment:
I know it doesn't really matter, but for cleanliness it would be great to decide on "yes, we are putting a period at the end of any list item that is a sentence or a reasonable compression of a sentence" or "no, we are not doing that".  Between the different short lists in this section of the diff, it's going back and forth between the two possibilities :-).

Reply:
The answer is supposed to be yes, but I failed to do so. Fixing.